addons/mail/static/src/js/views/activity/activity_renderer.js

```diff
@@ -208,7 +208,8 @@ var ActivityRenderer = AbstractRenderer.extend({
                 var def = widget.appendTo($td).then(function () {
                     // replace clock by closest deadline
                     var $date = $('<div class="o_closest_deadline">');
-                    var date = new Date(activity_group.o_closest_deadline);
+                    var tmp = activity_group.o_closest_deadline.split('-');
```

Comment:
Not a fan of a variable only called 'tmp' ... it's too vague

Reply:
renamed split_date